zhaquirks/tuya/ts0601_rcbo.py

```diff
@@ -248,10 +248,10 @@ def update_attribute(self, attr_name: str, value: Any) -> None:
         if attr_name == "active_power":
             apparent_power = self.get("apparent_power")
             if apparent_power:
-                power_factor = value / apparent_power * 1000
-                if power_factor > 1000:
-                    power_factor = 1000
-                super().update_attribute("power_factor", int(power_factor))
+                power_factor = value / apparent_power * 100
+                if power_factor > 100:
+                    power_factor = 100
+                super().update_attribute("power_factor", int(round(power_factor)))
```

Comment:
Minor, but I think just `round(power_factor)` should be enough, no `int(x)` needed in that case.

Reply:
Good point, thanks for highlighting that!